### ISSUE TITLE
[BUG] `-PoptionsFile` on gradle `hubRunFlow` task throws gradle error

### ISSUE THREAD

**Issue by mflatscher** [marklogic/marklogic-data-hub#4006]
#### The issue

**Short description of the problem:**
Setting `-Poptionsfile` on gradle task fails consistently.

**What behavior are you expecting?**
Ability to specify -PoptionsFile value

**What are the steps to reproduce the issue:**
1) With a gradle call as follows:

```
$ ./gradlew.bat -i hubRunFlow -Psteps=2 -PflowName="my-mapping-step" -PshowOptions=true -PoptionsFile=./optionsFile.json
```

2) And `optionsFile.json` as follows:

```
{
    "sourceQuery" : "cts.collectionQuery(\"some-collection\")"
}
```

3) I consistently get:

```
Execution failed for task ':hubRunFlow'.
> Cannot get property 'options' on extra properties extension as it does not exist
```
#### Tech details

**Which Operating System are you using?**
Windows 10

**Which version of MarkLogic are you using?**
10.0-3

**Which version of the Data Hub are you using?**
5.2.1

**Is this a QuickStart UI bug? If so, which browser are you using?**
N/A

**Comment by rahulvudutala** [comment]
@mflatscher Thank you for submitting the ticket. I have created an internal JIRA ticket to resolve the issue.
https://project.marklogic.com/jira/browse/DHFPROD-5083

**Comment by kkanthet** [comment]
Issue Fixed in 5.4.0 release
